src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,189 @@
+            fill="#FDFDFF"
+          />
+        </svg>
+      ),
+      link: "https://github.com/DaleStudy/leetcode-study",
+    },
+    {
+      label: "Youtube",
+      component: (
+        <svg
+          width="25"
+          height="18"
+          viewBox="0 0 25 18"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <path
+            fillRule="evenodd"
+            clipRule="evenodd"
+            d="M23.4017 1.76818C24.121 2.5016 24.3557 4.16708 24.3557 4.16708C24.3557 4.16708 24.5957 6.12336 24.5957 8.07889V9.9128C24.5957 11.8691 24.3557 13.8246 24.3557 13.8246C24.3557 13.8246 24.121 15.4901 23.4017 16.2235C22.5794 17.0911 21.6676 17.1808 21.1522 17.2314C21.0955 17.237 21.0436 17.2421 20.9972 17.2477C17.6387 17.4924 12.5957 17.5 12.5957 17.5C12.5957 17.5 6.3557 17.4426 4.4357 17.2568C4.34473 17.2397 4.23446 17.2264 4.10958 17.2113C3.5014 17.1378 2.54683 17.0225 1.78895 16.2235C1.0697 15.4901 0.835703 13.8246 0.835703 13.8246C0.835703 13.8246 0.595703 11.8691 0.595703 9.9128V8.07889C0.595703 6.12336 0.835703 4.16708 0.835703 4.16708C0.835703 4.16708 1.0697 2.5016 1.78895 1.76818C2.6131 0.899349 3.52576 0.810675 4.04118 0.760597C4.09707 0.755167 4.14828 0.750191 4.1942 0.744724C7.5527 0.5 12.5905 0.5 12.5905 0.5H12.601C12.601 0.5 17.6387 0.5 20.9972 0.744724C21.0431 0.750199 21.0944 0.755182 21.1503 0.760621C21.6653 0.810704 22.5784 0.89949 23.4017 1.76818ZM10.117 5.34387L10.1177 12.135L16.6015 8.75112L10.117 5.34387Z"
+            fill="#FDFDFF"
+          />
+        </svg>
+      ),
+      link: "https://www.youtube.com/@DaleSeo",
+    },
+  ];
+
+  return (
+    <footer className={containerStyle}>
+      <section className={sectionMenuStyle}>
+        <ul className={leftMenuStyle}>
+          {leftMenu.map(({ label, link }, i) => (
+            <li key={link + i}>
```

Comment:
`link` 값이 다 다른데, `link`로만으로도 충분히 `key`가 되지 않을까요?

```suggestion
            <li key={link}>
```

Reply:
link 값이 고유해서 key값으로 무관하겠네요!